get_authors.rb

```diff
@@ -20,9 +42,7 @@ def main
   files = Dir.glob File.join('*', '**', '*.md')
 
   files.each do |f|
-    author_mail = get_authors(f).each_slice(2).to_h.sort
-
-    author_md = author_mail.collect { |a, e| Markdown.mailto(a, e) }
+    author_md = Git.log(f).author_email.sort.map { |a, e| Markdown.mailto(a, e) }
```

Comment:
Interesting. Could you split this into multiple lines?

Reply:
Yes, I'll do that.

Reply:
Thanks!